deploy/database/updates/00462_tournament_01.sql

```diff
@@ -0,0 +1,30 @@
+ALTER TABLE tournament
+CHANGE is_double_elim tournament_type VARCHAR(50) NOT NULL;
+
+ALTER TABLE tournament
+CHANGE current_round round_number TINYINT UNSIGNED DEFAULT 1;
+
+ALTER TABLE tournament
+ADD tournament_state TINYINT UNSIGNED DEFAULT 10
+AFTER status_id;
+
+ALTER TABLE tournament
+ADD start_time TIMESTAMP DEFAULT 0
+AFTER id;
```

Comment:
Ugh, this is super fussy.  Why on earth not just drop the table and recreate it?

Reply:
At the end of the day, if the regen test passes (note to self: i haven't *done* the regen test yet), it passes, but what if you have to make a change that involves changing the table, and we have to keep stuff in sync.  I'd say switch to drop and recreate so we don't have to think about complicated things for no reason.

Reply:
Ah, reasonable request. I'll consolidate all these database updates into one file.

Reply:
Thanks!  I do think it'll save time over the course of our testing/iteration.